packages/docusaurus-theme-classic/src/theme/DocBreadcrumbs/index.tsx

```diff
@@ -71,13 +79,17 @@ export default function DocBreadcrumbs(): JSX.Element | null {
         styles.breadcrumbsContainer,
       )}
       aria-label="breadcrumbs">
-      <ul className="breadcrumbs">
+      <ul
```

Comment:
similarly, I didn't do it yet but this must be encapsulated under a `<Breadcrumbs>` component